src/scss/grommet-core/_objects.search.scss

```diff
@@ -141,14 +141,31 @@
     line-height: triple($inuit-base-spacing-unit);
 
     .search__input {
-      @include inuit-font-size($large-number-font-size, double($inuit-base-spacing-unit));
-      padding: halve($inuit-base-spacing-unit - $large-number-font-size - $input-border-width)
-        ($large-number-font-size - $input-border-width);
-      padding-right: (double($inuit-base-spacing-unit) - $input-border-width);
+      @include inuit-font-size($large-number-font-size * 1.5, double($inuit-base-spacing-unit));
+      padding: halve($inuit-base-spacing-unit);
+      padding-left: $inuit-base-spacing-unit * 3;
```

Comment:
I don't think we should shift the icon to the left for everyone. I think we should add a property to control this. I'd suggest iconAlign="start". The reason I think 'start' and 'end' is to be more semantic for right-to-left languages.